main/ertp/api/issuer.md

```diff
@@ -149,15 +148,15 @@ const quatloosPurse = quatloosIssuer.makeEmptyPurse();
 - `optAmount` `{Amount}` - Optional
 - Returns: `{Amount}`
 
-Burn (destroy) all of the digital assets in the `payment`
-and return an `amount` of what was burned. 
+Destroy all of the digital assets in the `payment`,
+make it unavailable for later use,
+and return an Amount of what was burned.
 
 `optAmount` is optional. If `optAmount` is present, 
 the code insists the `payment` balance is equal to `optAmount`, to prevent sending the wrong `payment`
 and other confusion.  
 
-If `payment` is a `promise` for a `payment`, the operation proceeds after the
-`promise` resolves.
+If `payment` is a promise, the operation proceeds after it resolves to a Payment.
```

Comment:
Presumably because there's no direct link between TS and this documentation. It does seem correct in the agoric-sdk source—[`issuer.burn` is typed as `IssuerBurn`](https://github.com/Agoric/agoric-sdk/blob/d5f9cadb93f7881bab173b926a149e58b6cee3d7/packages/ERTP/src/paymentLedger.js#L173-L174), which is [defined](https://github.com/Agoric/agoric-sdk/blob/1ee173920912812611d43f8a453faa920c65abac/packages/ERTP/src/types.js#L117-L132) to accept `ERef<Payment>` and optional `Pattern` and return `Promise<Amount>`. I've opened #666 to address here.